spec/service_consumers/pact_helper.rb

```diff
@@ -284,6 +284,18 @@ def url_encode(str)
     end
   end
 
+  provider_state "there are four content items with document_type 'topic'" do
+    set_up do
+      (1..4).each do |index|
+        FactoryGirl.create(:live_edition,
+                           title: "title_#{index}",
+                           document: FactoryGirl.create(:document),
+                           base_path: "/path_#{index}",
+                           document_type: "topic")
```

Comment:
Odd choice of document type, as the Taxonomy team is working on replacing the old topic taxonomy... but no real objection from me.